package/MDAnalysis/core/dummy.py

```diff
@@ -19,9 +19,7 @@
 # MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
-"""Mock Universe and Topology for testing purposes
-
-Avoids import of MDAnalysis at base level because of Issue #344
+"""Mock Universe and Topology generated from scratch with default values
```

Comment:
Seems sensible. Maybe `topology` makes more sense, then.
But we'd keep the reference to `make_Universe` at the module's top level, right?